subgraph/tests/referrer.test.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  assert,
+  describe,
+  test,
+  clearStore,
+  afterAll,
+  beforeAll,
+} from 'matchstick-as/assembly/index'
+import { createReferrerEvent } from './referrer-utils'
+import {
+  defaultMockAddress,
+  keyPrice,
+  lockAddress,
+  lockManagers,
+  nullAddress,
+  referrerFee,
+} from './constants'
+import { Address, BigInt, Bytes } from '@graphprotocol/graph-ts'
+import { handleReferrerFees } from '../src/public-lock'
+import { Lock } from '../generated/schema'
+
+describe('Referrer', () => {
```

Comment:
Added some unit tests too

Reply:
Great. Please add a test where a fee is set and then changed and make sure only a single entity exists

Reply:
New test added here with a single entity 🧼: https://github.com/unlock-protocol/unlock/pull/12596/commits/7f641bf0176fb07ecf071031760906f8b1906121